<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SelectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select...</string>
    </property>
   </widget>
   <widget class="QLabel" name="InstructionLabel_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Select the content folder to scan for .iso files: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScanButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="folderLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>539</width>
       <height>229</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Mountfoldershowerlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount folder path: </string>
    </property>
   </widget>
   <widget class="QLabel" name="MountPathLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MountPathButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select...</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>319</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>711</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Installed Products: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNativeInstall">
    <property name="title">
     <string>NativeInstall</string>
    </property>
   </widget>
   <addaction name="menuNativeInstall"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
